requirements.txt

```diff
@@ -1,15 +1,15 @@
 networkx==2.7.1
 numba==0.55.1
-numpy==1.21.5
+numpy==1.26.4
```

Comment:
Shouldn't this be further along?

Reply:
no clue really. I guess we could manually bump it, but also I suspect dependabot will do it for us?

Reply:
It's just the `==` that concerns me. I would manually replace all of the versions here with whatever gets installed via `setup.py`.